### DISCUSSION TITLE
Error Message

### DISCUSSION THREAD

**Discussion by Sakumaid111** [Official-Husko/NN-Downloader#28] [Q&A] [answered]
Hi i'm getting an Error message but can't tell why i'm getting this 

"An error occured while downloading from e621! Please report this. Exception: 'posts'"

**Comment by Official-Husko** [comment, marked as answer]
sounds like some api changes possibly? I will look into it. Make sure your config is configured properly. And next time please open an issue on the issues tab and not in the discussions.

**Reply by Sakumaid111** [reply]
"issues tab and not in the discussions"

Oh sorry about that ^^
